src/port1.0/porttrace.tcl

```diff
@@ -136,6 +136,7 @@ namespace eval porttrace {
 
         variable fifo
         variable fifo_mktemp_template
+        set PLACEHOLDER_DENY_XCODE no
```

Comment:
I'd prefer if this WIP change vanished from the history and you interactively rebased the final solution into this place.